### PR TITLE
Content update - Smart email check

### PR DESCRIPTION
**Pull request by thomasridd**
### What
Add message on check-content-details to pick up on simple email address typos

### How
- Use code ported from https://github.com/mailcheck/mailcheck/ to check based on word-distance 
- Use settings for most common UK and international domains
- Integrate into templates

### CODE REVIEW

**Review by niyioyelade — COMMENTED** [review]
Looks okay to me.

File: vulnerable_people_form/templates/check-contact-details.html [hunk 1]
```diff
@@ -7,6 +7,9 @@
   <h1 class="govuk-heading-l govuk-!-margin-top-0 govuk-!-margin-bottom-3">
     {{title_text}}
     </h1>
+    {% if alt_email_suggestion %}
+    <p class="govuk-body">Did you mean <strong>{{ alt_email_suggestion }}</strong></p>
```
Comment (sanisc) [inline vulnerable_people_form/templates/check-contact-details.html:11]: should this have a ? at the end?

Comment (sanisc) [reply vulnerable_people_form/templates/check-contact-details.html:11]: source :

https://www.figma.com/file/yqy8aBWZsu5LuYnmIaGTj7/Extremely-vulnerable-people-form?node-id=2862%3A0

Comment (thomasridd) [reply vulnerable_people_form/templates/check-contact-details.html:11]: yes it should

**Review by sanisc — APPROVED** [review]

**Pull request merged**